index.html

```diff
@@ -1,12 +1,454 @@
+
+                <div class="mech-section">
+                    <div class="mech-section__header-wrapper">
+                        <img class="mech-section__img" src="./images/vectors/Vector (4).png" alt="image">
+                        <h4 class="mech-section__txt">Digital markerting</h4>
+                    </div>
+                    <div class="mech-section__sub-text-wrap">
+                        <p class="mech-section__sub-text">Lorem ipsum dolor sit amet consectetur adipisicing.amet
+                            consectetur adipisicing</p>
+                    </div>
+                </div>
+
+                <div class="mech-section">
+                    <div class="mech-section__header-wrapper">
+                        <img class="mech-section__img" src="./images/vectors/clarity_users-line.png" alt="image">
+                        <h4 class="mech-section__txt">Digital markerting</h4>
+                    </div>
+                    <div class="mech-section__sub-text-wrap">
+                        <p class="mech-section__sub-text">Lorem ipsum dolor sit amet consectetur adipisicing.amet
+                            consectetur adipisicing</p>
+                    </div>
+                </div>
+
+                <div class="mech-section">
+                    <div class="mech-section__header-wrapper">
+                        <img class="mech-section__img" src="./images/vectors/Vector (6).png" alt="image">
+                        <h4 class="mech-section__txt">Digital markerting</h4>
+                    </div>
+                    <div class="mech-section__sub-text-wrap">
+                        <p class="mech-section__sub-text">Lorem ipsum dolor sit amet consectetur adipisicing.amet
+                            consectetur adipisicing</p>
+                    </div>
+                </div>
+
+                <div class="mech-section">
+                    <div class="mech-section__header-wrapper">
+                        <img class="mech-section__img" src="../images/vectors/Vector (7).png" alt="image">
+                        <h4 class="mech-section__txt">Digital markerting</h4>
+                    </div>
+                    <div class="mech-section__sub-text-wrap">
+                        <p class="mech-section__sub-text">Lorem ipsum dolor sit amet consectetur adipisicing.amet
+                            consectetur adipisicing</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- =========================================================================================== -->
+        <div class="container container--sm">
+            <div class="container__content">
+                <div class="page-text">
+                    <p class="p-text p-text--sm p-text-sm--normal">We are providing best <br> <span>business
+                            services.</span> </p>
+                </div>
+                <div class="page-text__sub-txt page-text__sub-txt--sm">
```

Comment:
`page-text__sub-txt` is positioned outside the Block `(page-text).` and also
`page-text__sub-txt--sm `is positioned outside the Block `(page-text)`.